src/utils/breakpoint/astBreakpointLocation.js

```diff
@@ -32,13 +35,17 @@ function findClosestScope(functions: Scope[], location: Location) {
 
 export async function getASTLocation(source: Source, location: Location) {
```

Comment:
side comment, but this reminds me of the code in `ast#setPreview` it'd be nice to move it out too